libs/librepcb/common/units/length.h

```diff
@@ -493,6 +493,14 @@ class Length {
     mNanometers *= rhs;
     return *this;
   }
+  Length& operator*=(int rhs) {
```

Comment:
The integer version of the operators is needed since otherwise, an ambiguity for type conversion is happening.

Reply:
Hmm if I remember correctly, I had good reasons to not implement operator overloads for `int` and `qreal` on `Length` and `Point`. But I'm really not sure anymore why exactly, I need to rethink about this :thinking: 

However, if you used `Toolbox::nearestPointOnLine()` for calculating the nearest point on a line, would you still need these additional operator overloads or would it solve this issue too?